docs/schedule-exceptions.adoc

```diff
@@ -0,0 +1,82 @@
+= Schedule exceptions
+
+Schedule exceptions describes any deviation from “regular schedule” generated from KOS data, i.e. state holiday, dean’s holiday, temporary change of room etc.
+Exceptions are applied to events (i.e. modifies its attributes) during automatic planning process.
```

Comment:
_automatic planning process_ kind of assumes you know Sirius internals (i.e. that there is a batch scheduling process) and it is not really helpful. It would be more helpful to say something like: _exceptions modify events' attributes fetched from KOS_.